source/broadband/webpa_notification.c

```diff
@@ -1041,12 +1041,50 @@ static void addNotifyMsgToQueue(NotifyData *notifyData)
 	WalPrint("*****Returned from addNotifyMsgToQueue*****\n");
 }
 
+/*
+ * @brief To handle sync notifications during factory reset
```

Comment:
Add a single line comment to delay sync notifications until FR cloud sync is acknowledged by cloud with Test and Set request for max wait period of 60s